lldb/test/Shell/SymbolFile/NativePDB/Inputs/namespace-access.lldbinit

```diff
@@ -0,0 +1,18 @@
+b main
+r
+
+type lookup S
+type lookup ::S
+type lookup Outer::S
+type lookup Outer::Inner1::S
+type lookup Inner1::S
+type lookup Outer::Inner1::Inner2::S
+type lookup Inner2::S
+type lookup Outer::Inner2::S
+type lookup Outer::A
+type lookup A
+type lookup ::A
+expr sizeof(S)
+expr sizeof(A)
+
+quit
```

Comment:
Lets put these commands into the test file itself. You can use `split-file` for this. For example: https://github.com/llvm/llvm-project/blob/81651e9fd0a744423fc0435f199ef79fb3a91f02/lldb/test/Shell/Settings/TestFrameFormatFunctionScopeObjC.test#L4